udata/core/post/api.py

```diff
@@ -72,7 +72,7 @@ class PostsAPI(API):
     def get(self):
         '''List all posts'''
         args = parser.parse_args()
-        return (Post.objects.order_by(args['sort'])
+        return (Post.objects.filter(published__ne=None).order_by(args['sort'])
```

Comment:
no `visible()` helper on Post model? Maybe it's time to add one :-)

Reply:
actually there is one already :)